ViewControllers/UIComponentsSignUpViewController.swift

```diff
@@ -18,60 +18,82 @@ class UIComponentsSignUpViewController: UIViewController {
     
     //MARK: - Variables
     let progress = Progress(totalUnitCount: Int64(Constant.FIVE))
+    var activeTextField : UITextView? = nil
```

Comment:
is it textview or textfield? one side you have written textField and other side you have written textView